src/dcm/agent/plugins/builtin/unmount_volume.py

```diff
@@ -48,22 +92,26 @@ def __init__(self, conf, job_id, items_map, name, arguments):
         super(UnmountVolume, self).__init__(
             conf, job_id, items_map, name, arguments)
 
-    def umount(self):
-        device_mappings = utils.get_device_mappings(self.conf)
-
-        for mapping in device_mappings:
-            d_id = mapping["device_id"]
-            mount_point = mapping["mount_point"]
-            if d_id == self.args.deviceId:
-                _unmount(self.conf, mount_point)
-                break
+    def unmount(self, mount_point):
+        _g_logger.info("Found a match in device mappings for %s...attempting unmount" % mount_point)
+        _unmount(self.conf, mount_point)
 
     def run(self):
+        device_mappings = utils.get_device_mappings(self.conf)
+        (platform, _) = utils.identify_platform(self.conf)
         try:
-            self.umount()
-            if self.args.encrypted:
-                close_encrypted_device(
-                    self.conf, self.args.deviceId)
+            for dev_map in device_mappings:
+                if dev_map["encrypted"] and self.args.deviceId in dev_map["device_id"]:
+                    deviceId = "mapper/es" + self.args.deviceId
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable "deviceId" to match the regular expression ^[_a-z][a-z0-9_]*$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.int.enstratius.com/coding_rules#rule_key=python%3AS117)